config/initializers/google_cloud_storage.rb

```diff
@@ -0,0 +1,10 @@
+require 'google/cloud/storage'
+
+CLOUD_STORAGE_API_JSON_KEY = ENV['CLOUD_STORAGE_API_JSON_KEY']
+
+if CLOUD_STORAGE_API_JSON_KEY
+  Google::Cloud::Storage.configure do |config|
+    config.project_id = 'teacher-vacancy-service'
```

Comment:
I appreciate this is unlikely to change, but can we please make it an ENV as well?